astroML/resample.py

```diff
@@ -42,7 +42,8 @@ def bootstrap(data, n_bootstraps, user_statistic, kwargs=None,
     data = np.asarray(data)
     if data.ndim != 1:
         n_samples = data.shape[0]
-        warnings.warn("bootstrap data are n-dimensional: assuming ordered n_samples by n_attributes")
+        warnings.warn("bootstrap data are n-dimensional: "
+                      "assuming ordered n_samples by n_attributes")
```

Comment:
this is fine, but I think we just need to allow to have a bit longer than 80char lines

Reply:
The line length is 101. I can `noqa` if you prefer that?